ldi-core/geojson-to-wkt/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/valueobjects/GeoType.java

```diff
@@ -6,23 +6,42 @@
 public enum GeoType {
 
 	// @formatter:off
-    POINT("https://purl.org/geojson/vocab#Point"),
-    LINESTRING("https://purl.org/geojson/vocab#LineString"),
-    POLYGON("https://purl.org/geojson/vocab#Polygon"),
-    MULTIPOINT("https://purl.org/geojson/vocab#MultiPoint"),
-    MULTIPOLYGON("https://purl.org/geojson/vocab#MultiPolygon"),
-    MULTILINESTRING("https://purl.org/geojson/vocab#MultiLineString"),
-    GEOMETRYCOLLECTION("https://purl.org/geojson/vocab#GeometryCollection");
+    POINT("https://purl.org/geojson/vocab#Point", "http://www.opengis.net/ont/sf#Point"),
+    LINESTRING("https://purl.org/geojson/vocab#LineString", "http://www.opengis.net/ont/sf#LineString"),
+    POLYGON("https://purl.org/geojson/vocab#Polygon", "http://www.opengis.net/ont/sf#Polygon"),
+    MULTIPOINT("https://purl.org/geojson/vocab#MultiPoint", "http://www.opengis.net/ont/sf#MultiPoint"),
+    MULTIPOLYGON("https://purl.org/geojson/vocab#MultiPolygon", "http://www.opengis.net/ont/sf#MultiPolygon"),
+    MULTILINESTRING("https://purl.org/geojson/vocab#MultiLineString", "http://www.opengis.net/ont/sf#MultiLineString"),
+    GEOMETRYCOLLECTION("https://purl.org/geojson/vocab#GeometryCollection", "http://www.opengis.net/ont/sf#GeometryCollection");
 	// @formatter:on
 
-	final String uri;
+	final String geoJsonUri;
+	final String sfUri;
```

Comment:
Could this name be renamed to something like `opengisSfUri` `simpleFeaturesUri` (this is where the acronym comes from), as this provides more context and makes it easier to lookup